src/api/admin.js

```diff
@@ -1,12 +1,66 @@
 import axios from '../axios';
 
 // authorization
+
+const createAdmin = (first_name, last_name, email, password, username) => {
+  const adminData = { first_name, last_name, email, password, username };
+  return axios
+    .post(`/admin`, adminData)
+    .then((res) => res.data)
+    .catch(function (error) {
+      console.log(error.toJSON());
+    });
+};
+
+const searchAdmin = (search, filters, page_size) => {
+  const searchAdminFilters = { search, filters, page_size };
+  return axios
+    .get(`/admin`, { params: { searchAdminFilters } })
+    .then((res) => res.data)
+    .catch(function (error) {
+      console.log(error.toJSON());
+    });
+};
+const getAdmins = () => {
+  return axios
+    .get(`/admin`)
+    .then((res) => res.data)
+    .catch(function (error) {
+      console.log(error.toJSON());
+    });
+};
 const getAdminById = (id) => {
-  return axios.get(`/admin/${id}`).then((res) => res.data);
+  return axios
+    .get(`/admin/${id}`)
+    .then((res) => res.data)
+    .catch(function (error) {
+      console.log(error.toJSON());
+    });
+};
+
+const updateAdmin = (id, first_name, last_name, email, password, username) => {
```

Comment:
change the function to have two param called id, body ex: updateAdmin = (id, body) => {}